### ISSUE TITLE
master broken on yarn compile-devtools

### ISSUE THREAD

**Issue by brendankenny** [GoogleChrome/lighthouse#3338]
This is due to Travis changing their version of the JDK and the devtools `compile_frontend.py` script being unable to parse the changed output of `java -version`.

Fix in progress here:
https://chromium-review.googlesource.com/c/chromium/src/+/663530

**Comment by paulirish** [comment]
fixed. now published to chrome-devtools-frontend@1.0.502075

**Comment by brendankenny** [comment]
still broken :(

```sh
Compiling frontend...
Compiling devtools_compatibility.js...
Verifying JSDoc comments...
JSDoc validator output:
Picked up _JAVA_OPTIONS: -Xmx2048m -Xms512m

devtools_compatibility.js compilation output:
Picked up _JAVA_OPTIONS: -Xmx2048m -Xms512m
0 error(s), 0 warning(s), 95.3% typed

devtools frontend compilation output:
Picked up _JAVA_OPTIONS: -Xmx2048m -Xms512m
0 error(s), 0 warning(s), 94.3% typed

ERRORS DETECTED
error Command failed with exit code 1.

The command "yarn compile-devtools" exited with 1.
```
